cortex/secondary/call_duration.py

```diff
@@ -2,40 +2,71 @@
 import numpy as np
 
 from ..feature_types import secondary_feature, log
-from ..raw.calls import calls
+from ..raw.telephony import telephony
 
 MS_IN_A_DAY = 86400000
+
+
 @secondary_feature(
     name='cortex.feature.call_duration',
-    dependencies=[calls]
+    dependencies=[telephony]
 )
-def call_duration(incoming=True, **kwargs):
+def call_duration(call_direction="all", **kwargs):
     """The time (in ms) spent talking on the phone.
 
     Args:
-        incoming (boolean): If True the duration of received calls is returned;
-            else the duration of sent calls is returned.
+        call_direction (string): If "incoming" the duration of received calls is returned;
+            if "outgoing" the duration of sent calls is returned;
+            if "all" the duration of all calls is returned.
+            Default parameter is "all".
         **kwargs:
             id (string): The participant's LAMP id. Required.
             start (int): The initial UNIX timestamp (in ms) of the window for which the feature
                 is being generated. Required.
             end (int): The last UNIX timestamp (in ms) of the window for which the feature
                 is being generated. Required.
+            incoming (boolean): Used to indicate direction of call. If not
```

Comment:
Please note somewhere in the docstring that "incoming" is a deprecated parameter but is left here for backwards compatibility.